src/main/java/README.md

```diff
@@ -0,0 +1 @@
+java files go here
```

Comment:
For the multi-module project, all this stub files should go in independent folders. I would also create instead the folder `core/src/main/java/org/htsjdk/core/package-info.java`:

```java
/**
 * Contains common code and interface/api definitions.
 */
package htsjdk.samtools;
```

And maybe at the root level (`core/README`):

```markdown
# HTSJDK-CORE

Contains common code and interface/api definitions.

Includes SAM/BAM, VCF and tribble support.

_Note: Should not include significant dependencies._
```